ERROR: ld.so: object '/usr/local/lib/python3.9/site-packages/scalene/libscalene.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
Scalene: could not find input file test/testme.py
